localstack/services/es/es_api.py

```diff
@@ -50,6 +59,7 @@ def get_domain_config_status():
 
 
 def get_domain_config(domain_name):
+    status = ES_DOMAINS.get(domain_name) or {}
```

Comment:
Thanks for this PR @mgagliardo 👍 

nitpick: We could assign the cluster config to a variable here:
```
cluster_cfg = status.get('ElasticsearchClusterConfig') or {}
default_cfg = DEFAULT_ES_CLUSTER_CONFIG
```
... and then use the variables below (to make the code a bit easier to read):
```
'DedicatedMasterCount': cluster_cfg.get('DedicatedMasterCount', default_cfg['DedicatedMasterCount']),
...
```